libs/net/entities/MediaType.cs

```diff
@@ -18,6 +18,12 @@ public class MediaType : BaseType<int>
     [Column("auto_transcribe")]
     public bool AutoTranscribe { get; set; }
 
+    /// <summary>
+    /// get/set - The type of entity that should be listed with this media type
+    /// </summary>
+    [Column("list_option")]
+    public ListOption ListOption { get; set; }
+
```

Comment:
- Just wondering if we need to have this enum set for all media types now, or whether it really only needs to be set in particular cases?  If it's only particular media types, then I would suggest tweaking the enum to the first enum value be `Default = 0` and then bump the other two values up by one.  Then this new column can be set to 0 by default.
- Should we have a migration script to set the initial value for all "known media types"  Or, if we follow my idea above, just the ones that need changing

Reply:
That's gonna be the case, if we keep the Source as a default, it will keep the actual data as it is in the moment of the migration. and them in my opinion, the change to other List Option should be done by the user.